tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -240,6 +240,11 @@ def build_arguments_parser():
         required=False,
         type=partial(get_path, is_directory=True)
     )
+    parser.add_argument(
+        '--subsample_size', help="number of samples to be used",
```

Comment:
it makes sence to have shortcut  `-ss` for this option for consistency with `convert_annotation` script

Reply:
done